rpc/README.md

```diff
@@ -1950,6 +1950,7 @@ Send new transaction into transaction pool.
 #### Parameters
 
     transaction - The transaction object, struct reference: https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0019-data-structures/0019-data-structures.md#Transaction
+    outputs_validator - Validates the transaction outputs before entering the tx-pool, an optional string parameter (enum: default | passthrough ), null means using default validator, passthrough means skipping outputs validation
```

Comment:
It worth documenting the behavior of the default validator.

Reply:
added a wiki page: https://github.com/nervosnetwork/ckb/wiki/Transaction-%C2%BB-Default-Outputs-Validator